packages/view/backend/sirius-components-view-emf/src/main/java/org/eclipse/sirius/components/view/emf/IViewRepresentationDescriptionSearchService.java

```diff
@@ -29,11 +28,7 @@ public interface IViewRepresentationDescriptionSearchService {
 
     Optional<RepresentationDescription> findById(IEditingContext editingContext, String representationDescriptionId);
 
-    Optional<NodeDescription> findViewNodeDescriptionById(IEditingContext editingContext, String nodeDescriptionId);
-
-    Optional<EdgeDescription> findViewEdgeDescriptionById(IEditingContext editingContext, String edgeDescriptionId);
-
-    Optional<FormElementDescription> findViewFormElementDescriptionById(IEditingContext editingContext, String formDescriptionId);
+    List<View> findByViewsBySourceId(IEditingContext editingContext, String sourceId);
```

Comment:
Methode name seems odd, we don't search by views.
Shouldn't it be `findViewsBySourceId` ?

Reply:
Yes indeed :+1: